.env.example

```
@@ -9,21 +9,23 @@
 
 # -----------------------------------------------------------------------------
-# GitHub Integration (required)
-# -----------------------------------------------------------------------------
-# Personal Access Token with 'repo' and 'workflow' scopes
-GITHUB_TOKEN=ghp_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-GITHUB_USER=your-github-username
-GITHUB_ORG=helios-platform-team
+# Gitea Integration (auto-configured by 'task setup:gitea-token')
+# -----------------------------------------------------------------------------
+GITEA_URL=http://localhost:3030
+GITEA_INTERNAL_URL=http://gitea-http.gitea.svc.cluster.local:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITEA_INTERNAL_URL key should go before the GITEA_URL key

(UnorderedKey)